### PR TITLE
Speed up Hydra Editor

### PR DESCRIPTION
**Pull request by terrellt**
This sacrifices the feature of hydra-editor to do dynamic look up of which edit
partial to use on a per-field basis. We don't use the feature now, don't intend
to, and this drops the render time from 10 seconds to ~.4. Tests don't get much quicker, though.

### CODE REVIEW

**Comment by terrellt** [comment]
@jechols We have the ability to filter out tests to only run on CI - do you think the Hydra-Editor feature spec is a valid case for that?

**Comment by jechols** [comment]
:+1: - Probably good for all feature specs, in fact, since we can run those manually as needed anyway

File: app/views/catalog/_search_form.html.erb [hunk 1]
```diff
@@ -20,4 +20,4 @@
   <% end %>
 
 
-<%= link_to 'More options', advanced_search_path(params), :class=>'advanced_search'%>
\ No newline at end of file
+<%= link_to 'More options', advanced_search_path(params.symbolize_keys), :class=>'advanced_search'%>
```
Comment (jechols) [inline app/views/catalog/_search_form.html.erb]: Totally not related to this PR, dude!

:+1: all the same (I'm assuming this fixes those awful deprecation warnings)

**Comment by jechols** [comment]
We should drop the capybara_login support file, too.  We probably don't want it available to accidentally use now that we have a faster option.

File: app/views/catalog/_search_form.html.erb (hunk 1 above)
Comment (terrellt) [inline app/views/catalog/_search_form.html.erb]: It does.

**Pull request merged**